src/clarity/clarity.module.ts

```diff
@@ -54,10 +54,13 @@ import {ClrResponsiveNavigationService} from "./nav/clrResponsiveNavigationServi
         TABS_DIRECTIVES,
         WIZARD_DIRECTIVES,
         ICON_DIRECTIVES
-    ],
-    providers: [
-        ClrResponsiveNavigationService
     ]
 })
 export class ClarityModule {
+    static forRoot() {
+        return {
+            ngModule: ClarityModule,
+            providers: [ ClrResponsiveNavigationService ]
```

Comment:
more details about `ScrollingService`: The service is used to support Modals in IE10. It adds the `.no-scrolling` class on `body` which sets `overflow: hidden` to avoid the content behind the backdrop from scrolling in IE10.